modules/installation-vsphere-installer-network-requirements.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vmc/installing-restricted-networks-vmc.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-installer-network-requirements_{context}"]
+= Network connectivity requirements
+
+You must configure the network connectivity between machines to allow {product-title} cluster components to communicate. Each machine must resolve the hostnames of all other machines in the cluster.
```

Comment:
For IPI, the hostname of each machines is resolved through static pod coredns in the cluster, no need additional configuration outside of the cluster, so it's better to remove statement "Each machine must resolve the hostnames of all other machines in the cluster." here

Reply:
Addressed